<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingDialog</class>
 <widget class="QDialog" name="settingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#settingDialog
{
border-image:url(:/img/main_background.jpeg);
}
#chooseDirectoryButton,#saveButton,#cancelButton
{
color:white;
font-size:18px;
font-weight:bold;
font-family:century gothic;
background:transparent;
}
#chooseDirectoryButton:hover,#saveButton:hover,#cancelButton:hover
{
	background:rgba(0,0,0,0.1);
}

QLineEdit#backupNameLineEdit,#backupDirectoryLineEdit,#passwordLineEdit
{
	background:transparent;
	border:none;
	color:white;
	border-bottom:1px solid white;
	font-family:times new roman;
	font-size:20px;
	font-weight:bold;
}

QLabel#label,#label_2,#label_4
{
	color:white;
	background:transparent;
	font-family:century gothic;
	font-size:20px;
	font-weight:bold;
}

QCheckBox#passwordCheckBox,#cloudCheckBox
{
	color:white;
	background:transparent;
	font-family:century gothic;
	font-size:20px;
	font-weight:bold;
}

QRadioButton#weekRadioButton,#onceRadioButton,#dayRadioButton
{
	color:white;
	background:transparent;
	font-family:century gothic;
	font-size:20px;
	font-weight:bold;
}

</string>
  </property>
  <widget class="QPushButton" name="chooseDirectoryButton">
   <property name="geometry">
    <rect>
     <x>468</x>
     <y>100</y>
     <width>85</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/browser.png</normaloff>:/img/browser.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>63</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>备份名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="backupNameLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>300</width>
     <height>39</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>63</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>备份到</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="backupDirectoryLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>300</width>
     <height>39</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>84</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>备份频率</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>201</width>
     <height>39</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>167</y>
     <width>317</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="onceRadioButton">
      <property name="text">
       <string>单次</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QRadioButton" name="dayRadioButton">
      <property name="text">
       <string>每日</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QRadioButton" name="weekRadioButton">
      <property name="text">
       <string>每周</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>222</y>
     <width>210</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QCheckBox" name="cloudCheckBox">
      <property name="text">
       <string>云备份</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="passwordCheckBox">
      <property name="text">
       <string>加密备份</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>保存</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>取消</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/img/cancel.png</normaloff>:/img/cancel.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
